### ISSUE TITLE
Bug 58071369: [React Native Gallery -> VirtualizedList]: Upon selecting any item from the "A VirtualizedList with single selection support", screen reader fails to announce the selected state information.

### ISSUE THREAD

**Issue by vineethkuttan** [microsoft/react-native-gallery#613]
Build Details:
OS Version: Dev (OS Build 27871.1000)
App: React Native Gallery 
App Version: 1.0.18.0
React Native Windows Version: 0.79.0

AT: Screen Reader
 
Pre-Requisite:
Turn on Narrator (Ctrl + Win + Enter).

Repro-Steps:
Launch the React Native Gallery app.
Navigate to ‘Collections' button present in left navigation pane and expand it.
Navigate to 'VirtualizedList' button and activate it.
Navigate to the item present under "A VirtualizedList with single selection support, heading" and activate it.
Listen to narrator announcement.
Actual Result:
Upon selecting any item from the "A VirtualizedList with single selection support", screen reader fails to announce the selected state information.
While navigating in between the different list items, narrator is not announcing the selected state of the list items. It simply announces as "Item 1, 1 of 50/ Item 2, 2 of 50/ and so on".

Expected Result:
Narrator is should the selected state of the list items. When user navigate in between the list items narrator should announce as "Item 1, 1 of 50, selected/ Item 2, 2 of 50, Unselected".

User Impact:
Users with visual impairments who rely on screen reader will not be able to know whether the button is selected/unselected if narrator fails to announce the state of the buttons. 

MAS Reference:
[MAS 4.1.2 – Name, Role, Value](https://aka.ms/MAS4.1.2)

Trap ID: 
[1.6 - Poor Grouping](https://aka.ms/PoorGrouping)

"Have feedback on bugs logged, please tag bug as “A11yRCA” and add your feedback in the comment section.”

**Comment by vineethkuttan** [comment]
#623 ,same changes should be made in src/examples/VirtualizedListExamplePage.tsx.

**Comment by vineethkuttan** [comment]
This issue is fixed in Fabric gallery.
